src/styles/blocks/days.scss

```diff
@@ -0,0 +1,57 @@
+.days {
+  padding: 10px;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
```

Comment:
$gap ?

Reply:
The instructions said so
![image](https://user-images.githubusercontent.com/109616961/187497147-71fe6dc9-8f00-4b0f-8d57-1ebfca5a6565.png)

Reply:
I meant use the $gap variable instead of 1px value

Reply:
Oh